packages/dso-toolkit/components/Voorbeelden/20-Toepassingen/60-Helpcentrum/Artikel.njk

```diff
@@ -0,0 +1,37 @@
+{% render '@header', {menu: menu, breadcrumbs: []} -%}
+<main>
+  <div class="row">
+    <div class="col-md-8">
+      <div class="dso-button-row">
+        {% render '@button', {type: 'button', modifier: 'dso-tertiary', label: 'Terug naar Terminologie', icon: 'chevron-left'} %}
+      </div>
+      <h1>Wat is een situatieschets?</h1>
+      <p>Een situatietekening is een tekening van een terrein met daarop aanwezige bebouwing. Hierbij wordt duidelijk hoe en waar gebouwen op de kavel staan. Ook aangelegen delen zoals de wegen, sloten en aanliggende bebouwing worden hierbij verwerkt. Soms wordt dit duidelijker gemaakt door het gebruik van kleuren.</p>
+      {% render '@image' %}
+      <span class="caption">Afb 1. Voorbeeld van een situatie tekening. Klik op de afbeelding voor een uitvergroting. </span>
+      <h2>Wat staat er op een situatietekening?</h2>
+      <p>Samenvattend is er op de situatietekening onder andere te zien:</p>
+      <ul>
```

Comment:
Hier hebben we ook een component voor, zie: https://dso-toolkit.nl/40.1.0/components/detail/lists.html

Reply:
aangepast